### ISSUE TITLE
Switch between A2DP and HSP/HFP input setting

### ISSUE THREAD

**Issue by edurenye** [kgshank/gse-sound-output-device-chooser#50]
With Bluetooth headsets in the settings you have an option to switch between A2DP (supports just output but has better quality, I use it for listening to music) and HSP/HFP (supports both input and output but with a lower quality of sound in the output, I use it for meetings).

With your plugin, when you select the input it switch automatically to HSP/HFP but then there is no way to go back to A2DP when you finished your meeting, so you have to enter the settings to do that.

So I have the need of continuously switching between those options inside the input settings.
Would be great that if you have an input with this options have a '>' that you can unfold and be able to switch between them.

The HSP/HFP quality is really bad for listening to music.

And I think there should be an option in settings to disable this feature if somebody does not what that feature, but should be enabled by default in my opinion.

**Comment by NickDeGraeve** [comment]
Any idea when, if ever, this is going to get in the extension?

**Comment by qlixed** [comment]
I was comment to ask for this, and just get the update notification, update and guess what? last stable version have this functionality. :smile:

**Comment by edurenye** [comment]
Yes, just updated and tried, seems to work fine.

Thank you very much!

**Comment by NickDeGraeve** [comment]
Where am I suppose to see this feature?
The plugin was updated this morning but I still can't find a way to switch profiles.

Some screenshots:
![Screenshot from 2020-05-25 07-54-41](https://user-images.githubusercontent.com/22236211/82783236-5fd30200-9e5e-11ea-9388-88008a38238b.png)

![Screenshot from 2020-05-25 07-55-27](https://user-images.githubusercontent.com/22236211/82783234-5ea1d500-9e5e-11ea-9fff-9f66042c1bab.png)

![Screenshot from 2020-05-25 07-55-01](https://user-images.githubusercontent.com/22236211/82783235-5f3a6b80-9e5e-11ea-88b5-7179568534b9.png)

**Comment by edurenye** [comment]
It does work for me on Ubuntu 20.04
![Screenshot from 2020-05-25 10-48-36](https://user-images.githubusercontent.com/6824576/82796729-535aa380-9e76-11ea-8117-d222e19fe801.png)

**Comment by edurenye** [comment]
Maybe has something to do with your headset, do this options appear on the regular 'Settings' > 'Sound'? If does not appear there, then is problem with your headset, not with this extension.

**Comment by NickDeGraeve** [comment]
I can select the profile from the settings menu:
![Screenshot from 2020-05-25 11-04-04](https://user-images.githubusercontent.com/22236211/82797855-f95add80-9e77-11ea-9335-c15b2b47818d.png)

I'm on Ubuntu 19.10. Maybe that's the problem?
After I upgraded to 20.04 I'll check again. Hopefully that's sometime this week.

**Comment by qlixed** [comment]
Volume mixer + Input chooser broke the profile selection, not sure why.
If I disable Volume Mixer the Options appears again.

**Comment by qlixed** [comment]
Quick update, the combo Volume mixer + Device chooser broke only if you select to put the Volume Mixer in the Status Meno, change that to any of the other options and all works again.

**Comment by NickDeGraeve** [comment]
I upgraded to 20.04 and now it seems to work.

Initially it still didn't, though. In the settings dialog I flipped some switches ("Hide selector..."  or "Enable new profile identification", I think) and all of a sudden the profiles appeared.
Then I thought something else was wrong as the other profile seemed disabled but that's just how it's visualized, I guess. Clicking on the "disabled" profile does the job.

I have to mention that after the upgrade to the newest version of the plugin but before the upgrade of Ubuntu the display manager crashed and restarted when I turned of my Bluetooth headset. When I disabled the plugin, it was ok. I didn't report it as I was about to upgrade Ubuntu anyway and it doesn't happen now.

But thanks.
